bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -15,8 +15,10 @@ unstable-ktls = ["s2n-tls-sys/unstable-ktls"]
 quic = ["s2n-tls-sys/quic"]
 fips = ["s2n-tls-sys/fips"]
 pq = ["s2n-tls-sys/pq"]
+testing = ["bytes"]
```

Comment:
Now that the old harness has been removed, we don't need the "bytes" dependency anymore.